### PR TITLE
[17.0][MIG] contract_operating_unit: Migration to 17.0

### PR DESCRIPTION
**Pull request by traitrantrobz**

### CODE REVIEW

**Comment by leemannd** [comment]
Hello @AaronHForgeFlow there is a migration of a module you created. Do you have time for a review?

**Review by leemannd — APPROVED** [review]

File: contract_operating_unit/tests/test_contract_operating_unit.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 class TestContractOperatingUnit(common.TransactionCase):
     def setUp(self):
```
Comment (hbrunn) [inline contract_operating_unit/tests/test_contract_operating_unit.py]: this should be converted to setUpClass

**Review by hbrunn — CHANGES_REQUESTED** [review]
the _prepare_invoice function needs migration.

I've added a test for this in the [v16 migration](https://github.com/OCA/operating-unit/pull/700), you should be able to just rebase your work on that

**Comment by yvaucher** [comment]
All dependencies have been merged, can your rebase and drop the last commit?

**Review by hbrunn — APPROVED** [review]

**Pull request merged**